dist/index.js

```diff
@@ -2563,7 +2563,9 @@ class BasicCredentialHandler {
         this.password = password;
```

Comment:
You can ignore all the changes in this file. It gets updated after running `npm run release`